src/clndr.js

```diff
@@ -387,6 +392,15 @@
         }
     };
 
+    Clndr.prototype.validateOptions = function() {
+        // Fix the week offset. It must be between 0 (Sunday) and
+        // 6 (Saturday).
+        if (this.options.weekOffset > 6 || this.options.weekOffset < 0) {
+            console.log('You have an invalid weekOffset');
```

Comment:
No need to block on this but for the sake of folks who are minifying/uglifying the source, it might be useful to make this a tad bit more descriptive. Something akin to...
```
console.warn(`clndr.js: An invalid offset ${this.options.weekOffset} was provided (must be 0 - 6); using 0 instead.`).
```

Reply:
I changed them to warn, and added that prefix to the messages. That's better, thanks.